tests/spanner_dbapi/test_types.py

```diff
@@ -17,8 +18,8 @@
 )
 from google.cloud.spanner_dbapi.utils import PeekIterator
 
-tzUTC = 0  # 0 hours offset from UTC
```

Comment:
Deleting this whitespace seems to be wrong. Black sets comments with two whitespaces